packages/api/src/utils.js

```diff
@@ -88,6 +143,9 @@ const decodeStateTransition = async (client, base64) => {
       decoded.nonce = parseInt(stateTransition.getNonce())
       decoded.outputScript = stateTransition.getOutputScript()?.toString('hex') ?? null
       decoded.coreFeePerByte = stateTransition.getCoreFeePerByte()
+      decoded.signature = stateTransition.getSignature()?.toString('hex') ?? null
```

Comment:
Signature Public Key Id is missing
User fee increase

Reply:
1/2 ✅